config/main.py

```diff
@@ -949,17 +950,18 @@ def _restart_services():
     wait_service_restart_finish('interfaces-config', last_interface_config_timestamp)
     wait_service_restart_finish('networking', last_networking_timestamp)
 
+    # Reload Monit configuration to pick up new hostname in case it changed
+    click.echo("Reloading Monit configuration ...")
+    clicommon.run_command(['sudo', 'monit', 'reload'])
+
     try:
         subprocess.check_call(['sudo', 'monit', 'status'], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
-        click.echo("Enabling container monitoring ...")
+        click.echo("Enabling container and routeCheck monitoring ...")
```

Comment:
After the route check is enabled at this point, as the links and sessions are still coming up, it may still cause same error if a route check happens, is that correct?

Reply:
@anamehra once we have start of monit probability of hitting ERR is less since it takes 5 min to initialize and then we have  15 min (3 cycle)  for any error to come. System should get stabilize by 15-20 min post monit restart.